src/OpenLoco/src/Objects/VehicleObject.h

```diff
@@ -158,6 +158,21 @@ namespace OpenLoco
     };
     OPENLOCO_ENABLE_ENUM_OPERATORS(BodySpriteFlags);
 
+    enum class CompanyColourType : uint8_t
```

Comment:
I think we should move this out of this header and make it standalone (probably under World). Its going to be used by company, gamecommands, vehicleobjects, etc.

Reply:
Should also use it for ChangeCompanyColourSchemeArgs::colourType

Reply:
This name feels a bit off, its used as an index to company->vehicleColours so wouldn't this be better called VehicleColourPreset or VehicleColourType?

Reply:
what would you name it if `Company::mainColour` was added to `company::vehicleColours` as the 0th index and then all the other indices were increased by 1?